appveyor.yml

```diff
@@ -0,0 +1,50 @@
+environment:
+  CSC_ENC_KEY: 
+    secure: <TODO>
+  CSC_KEY_PASSWORD:
+    secure: <TODO>
+  GH_TOKEN:
+    secure: <TODO>
+  APPVEYOR_RDP_PASSWORD:
+    secure: <TODO>
+  CSC_LINK: .\resource\certificates\win.p12
+
+image: Visual Studio 2017
+
+configuration:
+  - Release
+
+install:
+  - ps: Install-Product node 10
+  - cinst make
```

Comment:
Do we need to explicitly install MSYS/MINGW and Python, etc here as well? Unclear (currently unable to test this myself without access).